doc/quickstart.md

```diff
@@ -68,18 +68,18 @@ app.ddlog  db.url  deepdive.conf  input/  labeling/  mindbender/  README.md  udf
 ```
 
 ### 1. Load input
-
-You can find some of our sampled datasets under `input/`.
-You can also [download the full corpus][corpus], but let's proceed with the one that has 1000 sampled articles:
+First, you have to compile the DeepDive application using the following command:
 
 ```bash
-ln -s articles-1000.tsv.bz2 input/articles.tsv.bz2
+deepdive compile
 ```
+Once it has compiled with no error, you can run the following ```deepdive``` commands.
+
+You can find some of our sampled datasets under `input/`.
+You can also [download the full corpus][corpus], but let's proceed with the one that has 1000 sampled articles. Run the following command to load the sampled articles into DeepDive:
 ```bash
-deepdive do articles
+deepdive load articles input/articles-1000.tsv.bz2
```

Comment:
Maybe for a quicker quickstart, we could also just tell them to load the preprocessed sentences, not having to mention Java 8, etc.

```bash
deepdive do init/app
deepdive create table sentences
deepdive load sentences input/sentences-1000.tsv.bz2
deepdive mark done sentences
```

(Which doesn't look as simple as before though..)